src/config/mockStrategy.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Why is this added?

Reply:
It is to prevent eslint from throwing errors, because I am bringing in two packages (`passport-strategy` and `util`) that are in my `node_modules` but not in my `dependencies` or `devDependencies`, because they are installed by packages that I use. For instance, `passport-strategy` is installed by `passport`, and I think `util` is part of the `express` framework. But eslint will not permit me to use them unless they are listed in my dependencies.